github/gists.go

```diff
@@ -211,6 +234,29 @@ func (s *GistsService) Edit(id string, gist *Gist) (*Gist, *Response, error) {
 	return g, resp, err
 }
 
+// List gist commits.
+//
+// Github API docs: https://developer.github.com/v3/gists/#list-gist-commits
+
```

Comment:
In order for this comment to be considered as documentation by `godoc`, there can't be a blank line between it and the function. You should delete the blank line.

Source: https://golang.org/doc/effective_go.html#commentary

> Comments that appear before top-level declarations, with no intervening newlines, are extracted along with the declaration to serve as explanatory text for the item.